app/views/meals/new.html.erb

```diff
@@ -1,31 +1,63 @@
 <h1 class='m-4 text-center'>Log a Meal</h1>
-<h3 class='m-4 text-center'>Dishes in this Meal</h3>
-<% if @dishes.any? %>
-  <table class="table table-striped table-bordered cart-table">
-    <thead>
-      <tr>
-        <th scope="col">Dish Name</th>
-        <th scope="col">Edit</th>
-        <th scope="col">Delete</th>
-      </tr>
-    </thead>
-    <tbody>
-      <% @dishes.each do |dish| %>
-        <tr id= "dish-<%= dish.id %>">
-          <td><%= dish.name %></td>
-          <td><%= link_to 'Edit', edit_dish_path(dish), class: 'btn btn-sm btn-warning' %></td>
-          <td><%= button_to 'Delete', dish_path(dish), method: :delete, class: 'btn btn-sm btn-danger' %></td>
+<section class="current-dishes px-5">
+  <h3 class='m-4 text-center'>Dishes in this Meal</h3>
+  <% if @dishes.any? %>
+    <table class="table table-striped table-bordered cart-table">
+      <thead>
+        <tr>
+          <th scope="col">Dish Name</th>
+          <th scope="col" class='text-center'>Edit</th>
+          <th scope="col" class='text-center'>Delete</th>
         </tr>
-      <% end %>
-    </tbody>
-  </table>
-<% else %>
-  <p>There are currently no dishes in this meal</p>
-<% end %>
-<%= link_to 'Add a New Dish', new_dish_path, class:"btn btn-info" %>
-<%= form_for Meal.new do |f| %>
-  <%= f.label :title %>
-  <%= f.text_field :title %>
+      </thead>
+      <tbody>
+        <% @dishes.each do |dish| %>
+          <tr id= "dish-<%= dish.id %>">
+            <td><%= dish.name %></td>
+            <td class='text-center'><%= link_to 'Edit', edit_dish_path(dish), class: 'btn btn-sm btn-warning' %></td>
+            <td class='text-center'><%= button_to 'Delete', dish_path(dish), method: :delete, class: 'btn btn-sm btn-danger' %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  <% else %>
+    <p>There are currently no dishes in this meal</p>
+  <% end %>
+  <p class='text-center'>
+    <%= link_to 'Add a New Dish', new_dish_path, class:"btn btn-info m-4" %>
+  </p>
+</section>
+<section class='text-center m-4 border rounded p-4'>
+  <h4 class='mb-3'>Done Adding Dishes?</h4>
+  <%= form_for Meal.new do |f| %>
+    <%= f.text_field :title, placeholder: 'Meal Title' %>
 
-  <%= f.submit "Create Meal" %>
-<% end %>
+    <%= f.submit "Create Meal", class:"btn btn-sm btn-success" %>
+  <% end %>
+</section>
+
+<section class="dish-history px-5">
+  <h3 class='m-4 text-center'>Select From Previous Dishes</h3>
+  <% if current_user.dishes.any? %>
+    <table class="table table-striped table-bordered cart-table">
+      <thead>
+        <tr>
+          <th scope="col">Dish Name</th>
+          <th scope="col">Foods</th>
+          <th scope="col" class='text-center'>Add to Meal</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% current_user.dishes.each do |dish| %>
+          <tr id= "dish-<%= dish.id %>">
+            <td><%= dish.name %></td>
+            <td><%= dish.foods.pluck(:name)*", " %></td>
```

Comment:
We'll probably want to create a facade instead of using ActiveRecord in the view